pyetrade/order.py

```diff
@@ -520,44 +543,37 @@ def place_changed_equity_order(self, resp_format=None, **kwargs):
                 "No previewId given, previewing before placing order "
                 "because of an Etrade bug as of 1/1/2019"
             )
-            preview = self.preview_equity_order(resp_format, **kwargs)
-            if resp_format == "xml":
-                preview = jxmlease.parse(preview)
+            preview = self.preview_equity_order(**kwargs)
 
             if "Error" in preview:
-              LOGGER.error(preview)
-              raise Exception("Please check your order!")
+                LOGGER.error(preview)
+                raise Exception("Please check your order!")
 
-            kwargs["previewId"] = preview["PreviewOrderResponse"]["PreviewIds"][
-                "previewId"
-            ]
-            LOGGER.debug(
-                "Got a successful preview with previewId: %s", kwargs["previewId"]
-            )
+            kwargs["previewId"] = preview["PreviewOrderResponse"]["PreviewIds"]["previewId"]
+            LOGGER.debug("Got a successful preview with previewId: %s", kwargs["previewId"])
+
+        api_url = f'{self.base_url}/{kwargs["accountIdKey"]}/orders/{kwargs["orderId"]}/change/place'
 
-        api_url = self.base_url + "/" + kwargs["accountId"] + "/orders/"+kwargs["orderId"]+"/change/place"
         # payload creation
         payload = self.build_order_payload("PlaceOrderRequest", **kwargs)
 
-        return self.perform_request(self.session.put, resp_format, api_url, payload)
+        return self.perform_request(self.session.put, api_url, payload, "xml")
 
-    def cancel_order(self, account_id, order_num, resp_format=None):
+    def cancel_order(self, account_id_key: str, order_num: int) -> dict:

```

Comment:
We will need to add back the response format here
https://github.com/jessecooper/pyetrade/actions/runs/4473576965/jobs/7872705243

Reply:
fixed, see latest commit